### DISCUSSION TITLE
Interface: [::], port: 25672, protocol: clustering, purpose: inter-node and CLI tool communication

### DISCUSSION THREAD

**Discussion by asadhasnain** [rabbitmq/rabbitmq-server#9439] [Questions]
Is there a possibility to make RabbitMQ clustering to just bind to 127.0.0.1 instead of binding to all the available interfaces?

**Comment by lukebakken** [comment]
Yes, but obviously you won't be able to create a cluster.

https://blog.differentpla.net/blog/2023/04/01/inet-tcp-dist-localhost/

https://www.google.com/search?q=%22inet_dist_use_interface%22+rabbitmq
